src/libraries/System.Private.Xml/src/System/Xml/XmlConvert.cs

```diff
@@ -1657,5 +1657,241 @@ internal static Exception CreateInvalidNameCharException(string name, int index,
         {
             return CreateException(index == 0 ? SR.Xml_BadStartNameChar : SR.Xml_BadNameChar, XmlException.BuildCharExceptionArgs(name, index), exceptionType, 0, index + 1);
         }
+
+        public static bool TryFormat(bool value, Span<char> destination, out int charsWritten)
```

Comment:
I believe this one can do same optimization as in https://github.com/dotnet/runtime/pull/64782 but with correct casing. But I'd wait for API review first..